inference-engine/src/mkldnn_plugin/mkldnn_plugin.cpp

```diff
@@ -85,7 +85,9 @@ Engine::LoadExeNetworkImpl(const InferenceEngine::ICNNNetwork &network, const st
             return std::dynamic_pointer_cast<const ::ngraph::opset2::Gelu>(node) ||
                 std::dynamic_pointer_cast<const ::ngraph::opset2::BatchToSpace>(node) ||
                 std::dynamic_pointer_cast<const ::ngraph::opset2::SpaceToBatch>(node) ||
-                std::dynamic_pointer_cast<const ::ngraph::opset3::ShuffleChannels>(node);
+                std::dynamic_pointer_cast<const ::ngraph::opset3::ShuffleChannels>(node) ||
+                std::dynamic_pointer_cast<const ::ngraph::opset3::DepthToSpace>(node) ||
+                std::dynamic_pointer_cast<const ::ngraph::opset3::SpaceToDepth>(node);
```

Comment:
Since CPU plug-in  implementations of DepthToSpace and SpaceToDepth are not fixed yet I would advise to remove this lines. Otherwise you will not get green CI for now because of ngraph unit tests. I will disable decomposition for both layers together with CPU implementations fixes.

Reply:
Ok, I have reverted these changes.